x-pack/plugins/security_solution/scripts/endpoint/common/screen/column_layout_formatter.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import stripAnsi from 'strip-ansi';
-// eslint-disable-next-line import/no-extraneous-dependencies
+
 import ansiRegex from 'ansi-regex'; // its a dependency of `strip-ansi` so it should be fine
```

Comment:
Thanks!

> // its a dependency of `strip-ansi` so it should be fine

It's not, let's never do this again please (using transient-only dependencies in the code, scripts, or tests directly) :slightly_smiling_face: